### PR TITLE
python sdk readme guide

### PR DESCRIPTION
**Pull request by changlong-liu**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: documentation/code-gen/configure-python-sdk.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+``` yaml
+title: xxxxConfigurationClient
+description: xxxx Configuration Client
+openapi-type: arm
+tag: package-xxxx-xx-xx
+```
+~~~~
+
+### tag
+A tag can contains a bunch of swagger files which are used to generate the SDK. For Example:
```
Comment (akning-ms) [inline documentation/code-gen/configure-python-sdk.md:21]: Should we add some name rule for tag to unify it?

Comment (changlong-liu) [reply documentation/code-gen/configure-python-sdk.md:21]: yes, @akning-ms , I think we can recommend it as: package-yyyy-mm-dd[-xxxx].  where the xxxx can be 'preview' or 'only'.  I will add it in this PR.

File: documentation/code-gen/configure-python-sdk.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
+# Readme Configuration Guide for Python SDK
+This file describe how to configure readme files to make it avaiable for Python SDK code generation.
```
Comment (nickzhums) [inline documentation/code-gen/configure-python-sdk.md:2]: typo
"available"

File: documentation/code-gen/configure-python-sdk.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+Configure package title/description/tag.
+~~~~
+// file: readme.md
+
+``` yaml
+title: xxxxConfigurationClient
+description: xxxx Configuration Client
+openapi-type: arm
+tag: package-xxxx-xx-xx
+```
```
Comment (nickzhums) [inline documentation/code-gen/configure-python-sdk.md:17]: can we use a real example here?

File: documentation/code-gen/configure-python-sdk.md [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+~~~~
+
+### tag
+A tag can contains a bunch of swagger files which are used to generate the SDK.
```
Comment (nickzhums) [inline documentation/code-gen/configure-python-sdk.md:21]: do we need to explain benefits of using tag for service team?

File: documentation/code-gen/configure-python-sdk.md [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+python:
+  basic-setup-py: true
+  output-folder: $(python-sdks-folder)/appconfiguration/azure-mgmt-appconfiguration
+basic-setup-py: true
+output-folder: $(python-sdks-folder)/appconfiguration/azure-mgmt-appconfiguration
+```
+~~~
+
+## Multi-api
+If a multi-api package is need to be released, 'batch' should be used in the readme.python.md.
```
Comment (nickzhums) [inline documentation/code-gen/configure-python-sdk.md:154]: Maybe explain a little bit about multi-api to the service team. Basically it's a package that contains the API versions?

File: documentation/code-gen/configure-python-sdk.md [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
+    after_scripts:
+      - python ./scripts/multiapi_init_gen.py azure-mgmt-compute
+~~~
+
+
+## Run codegen
+After configure all the readme files, autorest can be used to generate SDK.
+### Track1
```
Comment (nickzhums) [inline documentation/code-gen/configure-python-sdk.md:221]: Describe Track 1 (Based on AutoRest version xxxxx that's going to be replaced by version yyyyy)

**Review by nickzhums — APPROVED** [review]

**Pull request merged**